cmd/oceantv/broadcast_states.go

```diff
@@ -716,6 +716,7 @@ func onFailureClosure(ctx *broadcastContext, cfg *BroadcastConfig, disableOnFirs
 		ctx.log("failed to start broadcast: %v", err)
 		ctx.bus.publish(startFailedEvent{})
 		try(ctx.man.Save(nil, func(_cfg *BroadcastConfig) {
+			_cfg.StartFailures++
 			const maxStartFailures = 3
```

Comment:
IMO, it is slightly easier to read if the const declarations are at the start of a block, i.e., before `_cfg.StartFailures++`

Reply:
fixed